tests/unit/modules/dcnm/fixtures/dcnm_net.json

```diff
@@ -0,0 +1,555 @@
+{
```

Comment:
Minor note but to be consistent can we use the full name for this file `dcnm_network`?

Reply:
Changes done as suggested.